package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "@contentful/rich-text-react-renderer": "^16.0.1",
-    "@justfixnyc/component-library": "0.53.9",
+    "@justfixnyc/component-library": "0.53.10",
```

Comment:
would you mind bumping this version to [54.1](https://www.npmjs.com/package/@justfixnyc/component-library) before you merge? I reverted the changes I made to the phone number TextInput and this new package reflects that.